### DISCUSSION TITLE
Amazon SNS Client and Dev Services

### DISCUSSION THREAD

**Discussion by akil-rails** [quarkusio/quarkus#33430] [Q&A]
I am following the getting started guide [here](https://quarkiverse.github.io/quarkiverse-docs/quarkus-amazon-services/dev/amazon-sns.html). I am using the dev services support mentioned [here](https://quarkiverse.github.io/quarkiverse-docs/quarkus-amazon-services/dev/dev-services.html). 
I write a script to create the SNS topic, specifying the script location with `quarkus.aws.devservices.localstack.init-scripts-folder`. 
The starter code however, requires the topicArn to publish an SNS message, and I am wondering how to get the topicARN (since the script runs in the container, how do I set the TOPIC_ARN for the app that runs on my local?)

[a comment by an automated account is not shown]

**Reply by jhask-slalom** [reply]
wondering the same thing
